api/params.go

```diff
@@ -31,6 +33,9 @@ type (
 
 	// A SlabID uniquely identifies a slab.
 	SlabID uint
+
+	// UploadID identifies an ongoing upload.
+	UploadID [8]byte
```

Comment:
Could make this `UploadID types.Specifier` to reuse some existing code.

Reply:
`types.Specifier` doesn't play nice with urls unfortunately